### ISSUE TITLE
The changes tab "Select All" checkbox's functionality is not represented to screenreaders

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6513]
The "select all" checkbox in the header of the changes list is not represented properly to screenreaders. They are able to explain its role and state but not what it's for. They do not announce that other controls are checked or unchecked based on this control's state.

**Comment by sergiou87** [comment]
Fixed in recent accessibility work ✨
